VideoLocker/res/values/styles.xml

```diff
@@ -37,6 +37,7 @@
         <item name="android:activatedBackgroundIndicator">@drawable/activated_item_selector</item>
         <item name="android:windowAnimationStyle">@style/EnterExitAnimation.Activity</item>
         <item name="android:dropDownListViewStyle">@style/edX.Widget.SpinnerDropDownListView</item>
+        <item name="android:textDirection">locale</item>
```

Comment:
I'm not sure why this is needed. The default `firstStrong` is supposed to fall back to the layout direction, which is already RTL